cmd/config_helper.go

```diff
@@ -92,9 +118,19 @@ func collectProviders(pkgSets ...[]*api.Package) map[string]string {
 	return providers
 }
 
-func collectDependencies(pkg string, requires []string, providers map[string]string, ignored map[string]bool) ([]string, error) {
+func collectDependencies(pkg string, requires []string, providers map[string]string, ignored map[string]bool, allPackages map[string]*bazeldnf.RPM) ([]string, error) {
```

Comment:
can we use something like https://pkg.go.dev/github.com/dominikbraun/graph to do a topological sort of the dependencies here and just get the result of that?